src/main/resources/site/site.xml

```diff
@@ -7,6 +7,11 @@
 		<field-set>
 			<label>General settings</label>
 			<items>
+				<input name="headless" type="CheckBox">
+					<label>Headless mode</label>
+					<help-text>If enabled, page processor will do nothing and its logic will be available in "metadata" service.</help-text>
+					<occurrences minimum="0" maximum="1"/>
+				</input>
```

Comment:
The frontend should disable page processing. As we do in the nextxp proxy: https://github.com/enonic/lib-nextxp/commit/fc72d3750740bee3914c4b8b96edc36ff92a03f8

Reply:
Why?

Reply:
You usually don't want the backend to modify your frontend html. (Currently headless often only does this inside xp/content studio rendering and not live)
The extension of this app is made to access the data for the frontend.

I guess it boils down to if you want to disable this on every app or one global place for all headless to disable. I think the fontend should deal with it.